app/views/admin/_header.html.erb

```diff
@@ -33,4 +33,5 @@
   <li><%= link_to ts('Tag Wrangling', key: 'header'), tag_wranglings_path %></li>
   <li><%= link_to ts('Locales', key: 'header'), locales_path %></li>
   <li><%= link_to ts('Activities', key: 'header'), admin_activities_path %></li>
+  <li><%= link_to ts('Manage API tokens', key: 'header'), admin_api_index_path %></li>
```

Comment:
'Tis I, the noble front end pseudo-Hound! Should be a capital "T" in "Tokens," please.